packages/bson-transpilers/test/yaml/bson-methods.yaml

```diff
@@ -31,6 +31,7 @@ tests:
                   'x': 1
                 }).scope
             object: "{'x': 1}"
+            php: (new \MongoDB\BSON\Javascript('test code', (object) ['x' => 1]))->getScope()
```

Comment:
Absolutely no need for `(object)` cast here. The Javascript constructor will always serialize the argument as a BSON document (see: [`php_phongo_javascript_init`](https://github.com/mongodb/mongo-php-driver/blob/1.14.0beta1/src/BSON/Javascript.c#L30)).

Reply:
Object case.

Reply:
OK to resolve.